### PR TITLE
Update financial and accounting documentation and strategy

### PR DESCRIPTION
**Pull request by choldgraf**
This PR fleshes out and updates several of our accounting and financial documentation. Here are the main goals:

- Update links to our budget projections and accounting data, and make them easier to quickly find.
- Add guiding principles and financial strategy information to help speed up decision making with financial implications.
- Update our "staffing costs" page to be a little bit more generic and focused on how we think about costs, rather than the specific scenario that was there before.
- Add a page about our cloud costs and how we reason about them
- Moved information about our credit cards into our finance/accounting section

I believe that this closes the following issues:

- closes https://github.com/2i2c-org/team-compass/issues/437

### CODE REVIEW

File: finance/accounting.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+(accounting:statements)=
+## Accounting statements
+
+We get monthly accounting reports from our fiscal sponsor, {term}`CS&S`.
+These are placed in a shared Google Drive that CS&S controls.
+
+For those with access, you can find the accounting statement folder here:
+
+```{button-link} https://drive.google.com/drive/u/1/folders/1vM_QX1J8GW5z8W5WemxhhVjcCS2kEovN
```
Comment (sgibson91) [inline finance/accounting.md:51]: I'm seeing the same `/u/1` part of this link that caused issues over in a recent JHub TC PR. Can we change the drive link to the share link of the folder please?

**Review by sgibson91 — APPROVED** [review]
I didn't go deep into the content here, because I trust Chris is the expert on this. Just a couple of points regarding Google Drive links that I've seen cause issues elsewhere.

File: finance/accounting.md (hunk 1 above)
Comment (choldgraf) [reply finance/accounting.md]: good catch

**Comment by choldgraf** [comment]
Thanks for the review @sgibson91 - I'll merge this in because I think it reflects our current (implicit) policy. We can revisit and improve this in subsequent PRs as we refine this strategy.

**Pull request merged**